server/svix-server/src/worker.rs

```diff
@@ -401,10 +401,15 @@ async fn dispatch(
         Err((attempt, err)) => {
             let attempt = attempt.insert(db).await?;
 
+            let retry_schedule = match endp.retry_schedule {
+                Some(retry_schedule) => retry_schedule.as_durations(),
+                None => cfg.retry_schedule.clone(),
```

Comment:
```suggestion
            let retry_schedule = match endp.retry_schedule {
                Some(retry_schedule) => retry_schedule.as_durations(),
                None => cfg.retry_schedule.clone(),
```
Don't clone, just borrow...

Reply:
This is trickier than it looks, at least to me. I'm not sure how to make `retry_schedule.to_durations()` -- or rather, `iter.collect()` -- return a reference.